covid-vaccine-tracker/covid-vaccine-tracker.js

```diff
@@ -0,0 +1,298 @@
+  DRC: "CD",
+  Montenegro: "ME",
+  Mayotte: "YT",
+  Kenya: "KE",
+  Vietnam: "VN",
+  Guatemala: "GT",
+  Venezuela: "VE",
+  Mali: "ML",
+  Paraguay: "PY",
+  ElSalvador: "SV",
+  Jamaica: "JM",
+  Tanzania: "TZ",
+  Martinique: "MQ",
+  Guadeloupe: "GP",
+  Rwanda: "RW",
+  Congo: "CG",
+  Brunei: "BN",
+  Somalia: "SO",
+  Gibraltar: "GI",
+  Cambodia: "KH",
+  Madagascar: "MG",
+  Gabon: "GA",
+  Myanmar: "MM",
+  Ethiopia: "ET",
+  Aruba: "AW",
+  FrenchGuiana: "GF",
+  Monaco: "MC",
+  Bermuda: "BM",
+  Togo: "TG",
+  EquatorialGuinea: "GQ",
+  Liberia: "LR",
+  Barbados: "BB",
+  Sudan: "SD",
+  Guyana: "GY",
+  Zambia: "ZM",
+  CaboVerde: "CV",
+  CaymanIslands: "KY",
+  Bahamas: "BS",
+  FrenchPolynesia: "PF",
+  Uganda: "UG",
+  Maldives: "MV",
+  Libya: "LY",
+  GuineaBissau: "GW",
+  Macao: "MO",
+  Haiti: "HT",
+  Syria: "SY",
+  Eritrea: "ER",
+  Mozambique: "MZ",
+  SaintMartin: "MF",
+  Benin: "BJ",
+  Chad: "TD",
+  Mongolia: "MN",
+  Nepal: "NP",
+  SierraLeone: "SL",
+  Zimbabwe: "ZW",
+  Angola: "AO",
+  Eswatini: "SZ",
+  Botswana: "BW",
+  Belize: "BZ",
+  NewCaledonia: "NC",
+  Malawi: "MW",
+  Fiji: "FJ",
+  Dominica: "DM",
+  Namibia: "NA",
+  SaintLucia: "LC",
+  Grenada: "GD",
+  CAR: "CF",
+  StVincentGrenadines: "VC",
+  TurksandCaicos: "TC",
+  FalklandIslands: "FK",
+  Greenland: "GL",
+  Montserrat: "MS",
+  Seychelles: "SC",
+  Suriname: "SR",
+  Nicaragua: "NI",
+  Gambia: "GM",
+  VaticanCity: "VA",
+  Mauritania: "MR",
+  PapuaNewGuinea: "PG",
+  StBarth: "BL",
+  Burundi: "BI",
+  Bhutan: "BT",
+  CaribbeanNetherlands: "BQ",
+  BritishVirginIslands: "VG",
+  SouthSudan: "SD",
+  Anguilla: "AI",
+  SaintPierreMiquelon: "PM",
+  Yemen: "YE",
+  China: "CN",
+};
+
+let dataList = document.getElementById("countries");
+let countriesInputList = document.getElementById("selectedCountry");
+let searchCounty = document.getElementById("search-button");
+let number = document.getElementById("number");
+const total = document.createElement("h1");
+
+searchCounty.addEventListener("click", onClick);
+function onClick() {
+  getVaccineData(countries[countriesInputList.value]);
+}
+window.onload = () => {
+  createDatalist();
+};
+
+function createDatalist() {
+  for (key in countries) {
+    createNewOption = () => {
+      const option = document.createElement("option");
+      option.value = key;
+      dataList.appendChild(option);
+    };
+    createNewOption();
+  }
+}
+let totalPerHundred ;
+function getVaccineData(countrycode) {
+  const url = `https://disease.sh/v3/covid-19/vaccine/coverage/countries/${countrycode}?lastdays=30&fullData=true`;
+  let i;
+  fetch(url)
+    .then((response) => response.json())
+    .then((data) => {
+      for (i = data.timeline.length - 1; i >= 0; i--) {
+        if (data.timeline[i].daily > 0) break;
+      }
+     
+      total.textContent = data.timeline[i].total;
+      number.appendChild(total);
+      totalPerHundred=(data.timeline[i].totalPerHundred)/100;
```

Comment:
As I test your Api their is a bug here the number of vaccinated people per hundred exceeds 100 in some countries and that's normal because some people take the vaccine multiple times but you need it not to exceed 100 in your code, I will show you where in the next comment 

Add some code lines to set this value to 1 if its >1